test/pack-native-to-base64-url/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "pack-native-to-base64-url",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "autoprefixer": "7.1.6",
+    "babel-core": "6.26.0",
+    "babel-loader": "7.1.2",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react-app": "^3.1.1",
+    "babel-runtime": "6.26.0",
+    "css-loader": "0.28.9",
+    "file-loader": "1.1.5",
+    "loader-runner": "^2.3.0",
+    "object-assign": "4.1.1",
+    "postcss-flexbugs-fixes": "3.2.0",
+    "postcss-loader": "2.0.8",
+    "promise": "8.0.1",
+    "raf": "3.4.0",
+    "react": "^16.2.0",
+    "react-dev-utils": "^5.0.0",
+    "react-dom": "^16.2.0",
+    "style-loader": "0.20.1",
+    "whatwg-fetch": "2.0.3"
+  }
+}
```

Comment:
Why do you need all this stuff?

Reply:
Don't need it at all, just recently learnt how to update snapshots. 😁
Will lighten the test case before getting ready to merge.